server/project-service/src/main/java/edu/illinois/ncsa/incore/service/project/models/WorkflowResource.java

```diff
@@ -1,13 +1,27 @@
 package edu.illinois.ncsa.incore.service.project.models;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.Date;
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
 public class WorkflowResource extends ProjectResource {
 
     // Enum for status
     public enum Type {
         workflow,
         execution
     }
-    private Type type;
+    public Type type;
+
```

Comment:
Would it make sense to add "finalized: boolean" field here to track whether this workflow is finalized or not?

Reply:
Added. Good suggestion!